app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
for the second problem, does it sound reasonable to change the "damage_fixed_reservation" to "repair_reservation", and the "damage_issued_reservation" to "rental_reservation"?